jacodb-ets/src/main/kotlin/org/jacodb/ets/model/EtsMethod.kt

```diff
@@ -26,10 +26,23 @@ interface EtsMethod : CommonMethod {
     val signature: EtsMethodSignature
     val localsCount: Int
     val cfg: EtsCfg
+    val modifiers: List<String>
```

Comment:
Add TODO about decorators.

Also, remove TODO about "modifiers" above the interface.